### PR TITLE
Add node brownout and ident event constants

### PR DESCRIPTION
**Pull request by atanisoft**
Currently I'm defining the brownout event in my esp32-s2 io code but would rather consume it directly from openlcb::Defs if possible. There are other well-known events but these two seemed to be more universally applicable to nodes.

### CODE REVIEW

File: src/openlcb/Defs.hxx [hunk 1]
```diff
@@ -246,6 +246,16 @@ struct Defs
     /// station may react to this by restoring locomotive speed settings.
     static constexpr uint64_t CLEAR_EMERGENCY_STOP_EVENT = 0x010000000000FFFCULL;
 
+    /// "Power supply brownout detected below minimum required by node"
```
Comment (balazsracz) [inline src/openlcb/Defs.hxx:249]: Isn't there a second of this with a similar flavor but different formulation? It might be worth adding both of them.

Comment (atanisoft) [reply src/openlcb/Defs.hxx:249]: Yes, there is a "below standard" one as well. I wasn't sure if it should be added or not but can add it.

File: src/openlcb/Defs.hxx [hunk 2]
```diff
@@ -246,6 +246,16 @@ struct Defs
...
+    /// This event can be generated when a node detects that it has
+    /// insufficient power for normal operations.
+    static constexpr uint64_t NODE_POWER_BROWNOUT_EVENT = 0x010000000000FFF1ULL;
+
+    /// "Ident button combination pressed"
+    /// This event can be generated by a node when it is instructed to generate
+    /// an identification event.
+    static constexpr uint64_t NODE_IDENT_BUTTON_EVENT = 0x010000000000FE01ULL;
```
Comment (balazsracz) [inline src/openlcb/Defs.hxx:257]: This is FE00 in the standard.

Comment (atanisoft) [reply src/openlcb/Defs.hxx:257]: Oops, will fix in a min. I read it off-by-one line.

**Review by balazsracz — APPROVED** [review]

**Pull request merged**